src/System.Net.Primitives/tests/FunctionalTests/System.Net.Primitives.Functional.Tests.csproj

```diff
@@ -33,13 +33,14 @@
     <Compile Include="IPAddressMappingTest.cs" />
     <Compile Include="IPEndPointTest.cs" />
     <Compile Include="NetworkCredentialTest.cs" />
-    <Compile Include="RequestCachePolicyTest.cs" Condition="'$(TargetGroup)' == ''" />
     <Compile Include="SocketAddressTest.cs" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetGroup)' == ''">
     <Compile Include="LoggingTest.cs" />
     <Compile Include="SerializationTest.cs" />
+    <Compile Include="RequestCachePolicyTest.cs" />
     <Compile Include="CookieContainerAddTest.cs" />
+    <Compile Include="NetworkCredentialTest.netstandard1.7.cs" />
```

Comment:
> $(TargetGroup) is being set as ' ' for netstandard1.7 and netcoreapp1.1

@safern that shouldn't be the case for netcoreapp1.1 (e.g. S.N.Security). If it is, it sounds like a bug.
I don't think we can build 1.3 anymore from this codebase (you've added the APIs without any `#if` pragmas).

What build error do you see without this separation?